test/Microsoft.SqlTools.ServiceLayer.UnitTests/TableDesigner/TableDesignerChangeInfoTest.cs

```diff
@@ -0,0 +1,30 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using Microsoft.SqlTools.ServiceLayer.TableDesigner.Contracts;
+using NUnit.Framework;
+using Newtonsoft.Json;
+
+namespace Microsoft.SqlTools.ServiceLayer.UnitTests.TableDesigner
+{
+    public class TableDesignerChangeInfoTest
+    {
+        [Test]
```

Comment:
Glad to see a test, but having a few more cases here would be grand.

(plus tests for the other parts of your code)

Reply:
so far majority of the the code I added are just entities to match the ADS side. I guess the only one other logic that is not test covered is adding new column, I'll have a test for that.